scss/mixins/_mixins.scss

```diff
@@ -239,10 +242,18 @@
   @include fd-active-pressed-selected() {
     @include fd-var-color("color", map-get($fd-colors-action-states, "selected"), --fd-color-action-selected);
     @include fd-var-color("border-bottom-color", map-get($fd-colors-action-states, "selected"), --fd-color-action-selected);
+    outline: none;
   }
   @include fd-disabled() {
     @include fd-var-color("color", map-get($fd-colors-action-states, "disabled"), --fd-color-action-disabled);
     @include fd-var-color("border-bottom-color", map-get($fd-colors-action-states, "disabled"), --fd-color-action-disabled);
+    @include fd-var-color("outline-color", fd-color-state("disabled", "action"), --fd-color-action-disabled);
+    cursor: not-allowed;
+  }
+  &:focus {
+    outline-style: dotted;
+    outline-width: 1px;
+    @include fd-var-color("outline-color", fd-color-state("hover", "action"), --fd-color-action-focus);
```

Comment:
If all the changes in this file are needed, I would expect these to be part of the link component's PR.  Once they are there, this PR can take advantage of those same styles.